examples/fluids/qfunctions/utils.h

```diff
@@ -0,0 +1,37 @@
+// Copyright (c) 2017-2022, Lawrence Livermore National Security, LLC and other CEED contributors.
+// All Rights Reserved. See the top-level LICENSE and NOTICE files for details.
+//
+// SPDX-License-Identifier: BSD-2-Clause
+//
+// This file is part of CEED:  http://github.com/ceed
+
+#ifndef utils_h
+#define utils_h
+
+#include <math.h>
+#include <ceed/ceed.h>
+
+CEED_QFUNCTION_HELPER CeedScalar Max(CeedScalar a, CeedScalar b) { return a < b ? b : a; }
+CEED_QFUNCTION_HELPER CeedScalar Min(CeedScalar a, CeedScalar b) { return a < b ? a : b; }
+
+CEED_QFUNCTION_HELPER CeedScalar Square(CeedScalar x) { return x*x; }
+CEED_QFUNCTION_HELPER CeedScalar Cube(CeedScalar x) { return x*x*x; }
+
+// @brief Dot product of 3 element vectors
+CEED_QFUNCTION_HELPER CeedScalar Dot3(const CeedScalar u[3],
+                                      const CeedScalar v[3]) {
+  return u[0]*v[0] + u[1]*v[1] + u[2]*v[2];
+}
```

Comment:
In Ratel I named all these helpers `RatelMax` and such just in case we include some other source with similarly named helpers to avoid colliding. Probably not needed here though